### PR TITLE
Upgrade mapnik to our @carto/mapnik

### PR DESCRIPTION
**Pull request by rafatower**
Pretty similar to https://github.com/CartoDB/abaculus/pull/5

cc/ @dgaubert

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ addons:
 sudo: false
 
 install:
- - npm install --fallback-to-build=false
+ - yarn install --fallback-to-build=false
```
Comment (dgaubert) [inline .travis.yml:17]: This is also failing because of our custom `mapnik` build??

File: package.json [hunk 2]
```diff
@@ -20,10 +20,10 @@
         "Blake Thompson <flippmoke>"
     ],
     "dependencies": {
-        "step"        : "~0.0.5",
-        "generic-pool": "~2.4.0",
-        "mapnik":       "3.5.14",
-        "mime"        : "~1.3.4",
+        "step"         : "~0.0.5",
+        "generic-pool" : "~2.4.0",
+        "@carto/mapnik": "~3.6.2-carto.0",
```
Comment (dgaubert) [inline package.json:25]: At some point, we'll have to have the same versioning approach for `tilelive-*`, what do you think?

File: test/support/assert.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ var path = require('path');
 var spawn = require('child_process').spawn;
 var exec = require('child_process').exec;
 var existsSync = require('fs').existsSync || require('path').existsSync;
-var mapnik = require('mapnik');
+var mapnik = require('@carto/mapnik');
```
Comment (dgaubert) [inline test/support/assert.js:7]: that looks awesome! ;)

**Review by dgaubert — COMMENTED** [review]
I left a couple of questions, nothing blocking

File: .travis.yml (hunk 1 above)
Comment (rafatower) [reply .travis.yml:17]: Yes, it is. To be honest, I don't know why that combination of npm version and our custom module fails.

File: package.json (hunk 2 above)
Comment (rafatower) [reply package.json:25]: Indeed, it is what I'm using now: https://github.com/CartoDB/tilelive-bridge/blob/2.5.1-cdb1/package.json#L2

Comment (dgaubert) [reply package.json:25]: I meant, the way we are versioning: [`2.5.1-cdb1`](https://github.com/CartoDB/tilelive-bridge/blob/2.5.1-cdb1/package.json#L3) vs  ~3.6.2-carto.0

Comment (rafatower) [reply package.json:25]: Oh, I did not get it before. I thought you were talking about namespaces and so on. I have no strong opinion. Those were mostly due to the infrastructure we have now for packaging mapnik and node-mapnik.

**Pull request merged**